docs/content/foundations/spacing/spacing-unit-tokens.mdx

```diff
@@ -0,0 +1,160 @@
+            additional_example_classes: 'height-4'
+          - token_name: '5'
+            multiple: '5'
+            calculated_value: '40px'
+            additional_example_classes: 'height-5'
+          - token_name: '6'
+            multiple: '6'
+            calculated_value: '48px'
+            additional_example_classes: 'height-6'
+          - token_name: '7'
+            multiple: '7'
+            calculated_value: '56px'
+            additional_example_classes: 'height-7'
+          - token_name: '8'
+            multiple: '8'
+            calculated_value: '64px'
+            additional_example_classes: 'height-8'
+          - token_name: '9'
+            multiple: '9'
+            calculated_value: '72px'
+            additional_example_classes: 'height-9'
+          - token_name: '10'
+            multiple: '10'
+            calculated_value: '80px'
+            additional_example_classes: 'height-10'
+          - token_name: '15'
+            multiple: '15'
+            calculated_value: '120px'
+            additional_example_classes: 'height-15'
+          - token_name: 'card'
+            multiple: '20'
+            calculated_value: '160px'
+          - token_name: 'card-lg'
+            multiple: '30'
+            calculated_value: '240px'
+          - token_name: 'mobile'
+            multiple: '40'
+            calculated_value: '320px'
+          - token_name: 'mobile-lg'
+            multiple: '60'
+            calculated_value: '480px'
+          - token_name: 'tablet'
+            multiple: '80'
+            calculated_value: '640px'
+          - token_name: 'tablet-lg'
+            multiple: '110'
+            calculated_value: '880px'
+          - token_name: 'desktop'
+            multiple: '128'
+            calculated_value: '1024px'
+          - token_name: 'desktop-lg'
+            multiple: '150'
+            calculated_value: '1200px'
+          - token_name: 'widescreen'
+            multiple: '175'
+            calculated_value: '1400px'
+          - token_name: '-1px, neg-1px'
+            multiple: '-'
+            calculated_value: '-1px'
+          - token_name: '-2px, neg-2px'
+            multiple: '-'
+            calculated_value: '-2px'
+          - token_name: '-0.5, neg-05'
+            multiple: '-.5'
+            calculated_value: '-4px'
+          - token_name: '-1, neg-1'
+            multiple: '-1'
+            calculated_value: '-8px'
+          - token_name: '-1.5, neg-105'
+            multiple: '-1.5'
+            calculated_value: '-12px'
+          - token_name: '-2, neg-2'
+            multiple: '-2'
+            calculated_value: '-16px'
+          - token_name: '-2.5, neg-205'
+            multiple: '-2.5'
+            calculated_value: '-20px'
+          - token_name: '-3 ,neg-3'
+            multiple: '-3'
+            calculated_value: '-24px'
+          - token_name: '-4, neg-4'
+            multiple: '-4'
+            calculated_value: '-32px'
+          - token_name: ', neg-5'
```

Comment:
@karlikpj I assume this is a mistake and USWDS did not exclude `"-5"` as a valid unit.